default/it/help_opera_houses.md

```diff
@@ -96,15 +96,15 @@
  Városi Színház  
  Várszínház (Buda) |
 | Caserta:  
-  
-  
-  
-  
-  
-  
-  
+
```

Comment:
These blank spaces should be removed

Reply:
Will do that in a separate phase. I just changed the name here. Hopefully this file will disappear completely in the future (it is a "joint file" to be shared)